### PR TITLE
Use uncertainity forecast instead of solve

### PR DESCRIPTION
**Pull request by Vaibhavdixit02**

### CODE REVIEW

**Comment by ChrisRackauckas** [comment]
"Instead of", instead can you add it in addition?

Also, this has merge conflicts.

File: docs/src/Scenario1/Evaluation_Scenario_1.md [hunk 1]
```diff
@@ -14,21 +14,23 @@ which we will use later.
 ```@example scenario1
 using EasyModelAnalysis, LinearAlgebra
 using EasyModelAnalysis.ModelingToolkit: toparam
-using EasyModelAnalysis.ModelingToolkit.Symbolics: FnType, variables
+using EasyModelAnalysis.ModelingToolkit.Symbolics: FnType, variable, variables
```
Comment (YingboMa) [inline docs/src/Scenario1/Evaluation_Scenario_1.md:17]: Why do you need to import `variable`?

**Comment by Vaibhavdixit02** [comment]
Depends on https://github.com/SciML/EasyModelAnalysis.jl/pull/139

File: docs/src/Scenario2/Evaluation_Scenario_2.md [hunk 2]
```diff
@@ -264,7 +304,7 @@ plot(sol_opt_p, idxs = [threshold_observable], lab = "total infected", leg = :bo
 
 ### Form SIDARTHE-V model
 
-This forms SIDARTHE-V by manually adding the V state and vax transition. It compares the models via maximum common subacset, plotting the common subgraph (the original SIDARTHE), the negation (the new transition and vax state), and the complement (the new transition from susceptible to vax).
+This forms SIDARTHE-V by manually adding the V state and vax transition. It compares the models via maximum common subset, plotting the common subgraph (the original SIDARTHE), the negation (the new transition and vax state), and the complement (the new transition from susceptible to vax).
```
Comment (ChrisRackauckas) [inline docs/src/Scenario2/Evaluation_Scenario_2.md:307]:
```suggestion
This forms SIDARTHE-V by manually adding the V state and vax transition. It compares the models via maximum common subacset, plotting the common subgraph (the original SIDARTHE), the negation (the new transition and vax state), and the complement (the new transition from susceptible to vax).
```

why was that changed?
